app/src/main/java/com/kickstarter/ui/viewholders/HorizontalRewardViewHolder.kt

```diff
@@ -0,0 +1,159 @@
+import kotlinx.android.synthetic.main.item_reward.view.*
+
+class HorizontalRewardViewHolder(private val view: View) : KSViewHolder(view) {
+
+    private val ksString = environment().ksString()
+    private var viewModel = HorizontalRewardViewHolderViewModel.ViewModel(environment())
+
+    private val currencyConversionString = context().getString(R.string.About_reward_amount)
+    private val pledgeRewardCurrencyOrMoreString = context().getString(R.string.rewards_title_pledge_reward_currency_or_more)
+    private val projectBackButtonString =  context().getString(R.string.project_back_button)
+    private val remainingRewardsString = context().getString(R.string.Left_count_left_few)
+
+    init {
+
+        this.viewModel.outputs.conversionTextViewIsGone()
+                .compose(bindToLifecycle())
+                .compose(observeForUI())
+                .subscribe(ViewUtils.setGone(view.horizontal_reward_usd_conversion_text_view))
+
+        this.viewModel.outputs.conversionTextViewText()
+                .compose(bindToLifecycle())
+                .compose(observeForUI())
+                .subscribe { setConversionTextView(it) }
+
+        this.viewModel.outputs.descriptionTextViewText()
+                .compose(bindToLifecycle())
+                .compose(observeForUI())
+                .subscribe { view.horizontal_reward_description_text_view.text = it }
+
+        this.viewModel.outputs.isClickable()
+                .compose(bindToLifecycle())
+                .compose(observeForUI())
+                .subscribe { view.horizontal_reward_pledge_button.isClickable = it }
+
+        this.viewModel.outputs.limitAndRemainingTextViewIsGone()
+                .compose(bindToLifecycle())
+                .compose(observeForUI())
+                .subscribe(ViewUtils.setGone(view.horizontal_reward_remaining_text_view))
+
+        this.viewModel.outputs.limitAndRemainingTextViewText()
+                .compose(bindToLifecycle())
+                .compose(observeForUI())
+                .subscribe {
+                    setRemainingRewardsTextView(it.second)
+                }
+
+        this.viewModel.outputs.minimumTextViewText()
+                .compose(bindToLifecycle())
+                .compose(observeForUI())
+                .subscribe {
+                    view.horizontal_reward_minimum_text_view.text = it
+                    setMinimumTextView(it)
+                }
+
+        this.viewModel.outputs.reward()
+                .compose(bindToLifecycle())
+                .compose(observeForUI())
+                .subscribe { view.horizontal_reward_ending_text_view.text = formattedDeadlineString(it) }
+
+        this.viewModel.outputs.rewardEndDateSectionIsGone()
+                .compose(bindToLifecycle())
+                .compose(observeForUI())
+                .subscribe {
+                    ViewUtils.setGone(view.horizontal_reward_ending_text_view, it) }
+
+        this.viewModel.outputs.titleTextViewText()
```

Comment:
could this just be `title`?

Reply:
This name is following the same convention of the other strings, like `backersTextViewText`, `converstionTextViewText`... should we change all of them to a shorter version or would it be ok to just change the `title`?

Reply:
yeah it seems super redundant to add `TextViewText` to every string